packages/fds-components/src/style/interactive/toaster.css

```diff
@@ -47,3 +53,8 @@ body {
   opacity: 0.01; /* removed from DOM on duration complete */
   transform: translateY(100px);
 }
+
+/* Targeting a nested item isn't ideal, but material may block us from targeting this directly */
+.toast-action span {
+  line-height: 1;
+}
```

Comment:
I'm going to sort of call this out in advance for your convenience :)

Whatever line-height that is on here by default (1.75) makes the button appear "slightly" out of line with things. This is a material button so I'm not sure I have access to this span's line-height (though totally let me know).

Reply:
this is good for now - I imagine we'll want to replace this with one of our own `Button` components when we're ready.